guide/src/format/mdbook.md

```diff
@@ -82,6 +82,12 @@ These use the same attributes as [rustdoc attributes], with a few additions:
 [editor]: theme/editor.md
 [`rust.edition`]: configuration/general.md#rust-options
 
+## Code block attributes
+
+The following attributes can be used on *any* code block to toggle specific features: 
+
+* `nocopy` - Disables the "Copy to Clipboard" button
+
```

Comment:
Since code blocks arguably overlap with syntax highlighting, maybe add a line pointing at that?

```suggestion

See also [syntax highlighting](theme/syntax-highlighting.md).
```